contracts/adapters/yearn/YFIAdapter.sol

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2020 Zerion Inc. <https://zerion.io>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.5;
+pragma experimental ABIEncoderV2;
+
+import { ERC20 } from "../../ERC20.sol";
+import { ProtocolAdapter } from "../ProtocolAdapter.sol";
+
+
+
+
+
+
+/**
+ * @dev Yearn YFI Rewards Token Interface.
+ * Only the functions required for YFIAdapter contract are added.
+ * The YearnRewards contract is available here
+ * 0xcc9EFea3ac5Df6AD6A656235Ef955fBfEF65B862
+ */
+
+/*
+interface YearnRewards {
+    function claimable(address) external view returns (uint);
+}
+*/
+
+/**
+ * @dev YearnRewards contract interface.
+ * Only the functions required for YFIAdapter contract are added.
+ * The YearnRewards Curve Pool contract is available here
+ * 0xcc9EFea3ac5Df6AD6A656235Ef955fBfEF65B862
+ * The YearnRewards Balancer Pool contract is available here
+ * 0x033E52f513F9B98e129381c6708F9faA2DEE5db5
+ */
+
+interface YearnRewards {
+    function earned(address) external view returns (uint256);
+}
```

Comment:
```suggestion
/**
 * @dev StakingRewards contract interface.
 * Only the functions required for YearnStakingV1Adapter contract are added.
 * The StakingRewards contract is available here
 * github.com/Synthetixio/synthetix/blob/master/contracts/StakingRewards.sol.
 */
interface StakingRewards {
    function earned(address) external view returns (uint256);
}
```

Reply:
this interface is enough as they use Synthetix's contracts

Reply:
Thanks - I thought I had to use the specific interface/contract name of the contract pointed to in the 'address internal constant' declaration